code/modules/atmospherics/machinery/components/unary_devices/vent_pump.dm

```diff
@@ -320,6 +320,14 @@
 	pipe_vision_img.plane = ABOVE_HUD_PLANE
 	playsound(loc, 'sound/weapons/bladeslice.ogg', 100, 1)
 
+
+/obj/machinery/atmospherics/components/unary/vent_pump/AltClick(mob/living/user)
```

Comment:
```suggestion
/obj/machinery/atmospherics/components/unary/vent_pump/AltClick(mob/user)
```
And define `can_ventcrawl()` as a mob proc, else this has the risk of causing runtimes.

Reply:
Actually scratch that. `handle_ventcrawl()` already checks for `can_ventcrawl()`. Redefine that one to the mob level.